README.md

```diff
@@ -96,6 +99,13 @@ can be used for composing other streams that depend on it. This observable is di
 when its `refCount` goes down to zero.
 
 ### distinctShareReplay
+```ts
+const activePlanetName$ = planet$.pipe(
+  filter(planet => planet.isActive),
+  map(planet => planet.name),
+  distinctShareReplay()
+)
+```
```

Comment:
This is awesome! I wanted to start each section with a clear and concise example, but then I got lazy. I like this.